app/src/substrate_types_V1.c

```diff
@@ -3933,6 +3932,7 @@ parser_error_t _toStringPortfolioKind_V1(
     switch (v->value) {
     case 0:
         snprintf(outValue, outValueLen, "Default");
+        (*pageCount)++;
```

Comment:
Might be better to add before the switch
` *pageCount = 1;`

So if in the future another case is added with only one page, the default value of pages is already there.